src/cli-flags-helper.js

```diff
@@ -0,0 +1,39 @@
+const targetList = new Set([
+  "acorn",
+  "babel",
+  "babylon",
+  "buble",
+  "chai",
+  "coffeescript",
+  "espree",
+  "esprima",
+  "jshint",
+  "lebab",
+  "prepack",
+  "prettier",
+  "source-map",
+  "typescript",
+  "uglify-es",
+  "uglify-js"
+]);
+
+function getOnlyFlag(argv) {
```

Comment:
The `argv` argument is not used. Either remove it, or replace the use of `process.argv` below with `argv`.

Reply:
Oops! My bad! Sorry..